src/index.html

```diff
@@ -240,29 +240,29 @@ <h4>Converting genetic variation data to VCF</h4>
-                            This metadata is described in an Excel template that can be found <a href="files/EVA_Submission_template.V1.1.1.xlsx">here</a>. Please also see <a href="files/EVA_Submission_template.V1.1.1_mockup.xlsx">here</a> for a mocked up version of this template that has been completed for a fictional study.
+                            Submissions to the EVA consist of VCF file(s), any associated data file(s), and metadata that describe sample(s) and analysis that produced the variant and/or genotype call(s).
+                            This metadata is described in an Excel template that can be found <a href="files/EVA_Submission_template.V1.1.1.xlsx">here</a>. Please also see <a href="files/EVA_Submission_template.V1.1.1_mockup.xlsx">here</a> for a mocked up version of this template that has been completed for a fictional study. Please note that the template requires the submitter to fill in some personal data, which will be used as described in <a href="https://www.ebi.ac.uk/data-protection/privacy-notice/eva-submission-service">our privacy notice</a>.
                         </p>
                         <p>
                             VCF file(s) submitted to the EVA must be truly valid a 4.X version of the file format <a href="http://samtools.github.io/hts-specs/VCFv4.3.pdf" target="_blank">specification</a>.
                             Files can be validated prior to submission using our validation suite that is available via our <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a>.
                         </p>
                     </div>
 
-                    <h4><span class="icon icon-generic" data-icon="C" aria-hidden="true"></span> Contact</h4>
+                    <h4 name="submit-contact"><span class="icon icon-generic" data-icon="C" aria-hidden="true"></span> Contact</h4>
                     <div>
                         <p>Contact  <a href="mailto:eva-helpdesk@ebi.ac.uk" target="_top">eva-helpdesk@ebi.ac.uk</a> in order to provide details of your submission. You shall receive a custom private FTP location for you to deposit files.</p>
                     </div>
 
-                    <h4><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
+                    <h4 name="submit-submit"><span class="icon icon-functional" data-icon="_"></span> Submit</h4>
                     <div>
                         <p>Upload your VCF file(s), associated data file(s) and EVA metadata template to your private EVA FTP location.</p>
                     </div>
 
-                    <h4><span class="icon icon-functional" data-icon="k" aria-hidden="true"></span> Receive</h4>
+                    <h4 name="submit-receive"><span class="icon icon-functional" data-icon="k" aria-hidden="true"></span> Receive</h4>

```

Comment:
is this for linking to subsections of the page?

Reply:
Yes, this allows linking with https://www.ebi.ac.uk/eva/?Submit-Data#submit-receive

Even if everything is in a single HTML, we might be able to rename to just receive if you think it looks better.

Reply:
I don't mind. it's fine like this